### PR TITLE
Force sympy 1.0

### PR DESCRIPTION
**Pull request by mloubout**
This PR is not a solution but a hack so that people don't have problem installing devito while we figure out how to handle sympy 1.1

Only change the setup to force 1.0

### CODE REVIEW

**Comment by mloubout** [comment]
Oups, TRAVIS uses requirement not setup, will fix it.

At least TRAVIS shows the problem with `sympy 1.1`

File: setup.py [hunk 1]
```diff
@@ -20,6 +20,6 @@
       author_email='opesci@imperial.ac.uk',
       license='MIT',
       packages=find_packages(exclude=['docs', 'examples', 'tests']),
-      install_requires=['numpy', 'sympy', 'mpmath', 'cgen', 'codepy',
+      install_requires=['numpy', 'sympy==1.0', 'mpmath', 'cgen', 'codepy',
```
Comment (FabioLuporini) [inline setup.py]: Shouldn't this list match that in requirements.txt? Or can we just drop this @mlange05

**Comment by FabioLuporini** [comment]
OK. @mlange05 I think Mathias is right and we should merge this. We should also add an issue though

File: setup.py (hunk 1 above)
Comment (mloubout) [inline setup.py]: Also is in the requirements

**Comment by FabioLuporini** [comment]
OK, merging...

**Comment by FabioLuporini** [comment]
Merged.

**Pull request merged**